src/main/java/thestonedturtle/runiterocks/RuniteRocksPanel.java

```diff
@@ -316,7 +316,7 @@ public void mousePressed(MouseEvent mouseEvent)
 	 */
 	private TableRow buildRow(World world, boolean current, RuniteRock rock)
 	{
-		TableRow row = new TableRow(world, rock, plugin::hopToWorld, plugin::removeRock, plugin.config.respawnCounter(), plugin.config.visitCounter());
+		TableRow row = new TableRow(world, rock, plugin::hopToWorld, plugin::removeRock, plugin.config.respawnCounter(), plugin.config.visitCounter(), plugin);
```

Comment:
Just pass the config here instead of passing the entire config.

You could also make a new method in the plugin and pass the runnable here, but I think ensuring they double click on the same row would be harder inside the plugin